modules/nodes-cluster-worker-latency-profiles-about.adoc

```diff
@@ -8,33 +8,35 @@
 [id="nodes-cluster-worker-latency-profiles-about_{context}"]
 = Understanding worker latency profiles
 
-Worker latency profiles are multiple sets of carefully-tuned values for the `node-status-update-frequency`, `node-monitor-grace-period`, `default-not-ready-toleration-seconds` and `default-unreachable-toleration-seconds` parameters. These parameters let you control the reaction of the cluster to latency issues without needing to determine the best values manually.
+Worker latency profiles are four different categories of carefully-tuned parameters. The four parameters which implement these values are `node-status-update-frequency`, `node-monitor-grace-period`, `default-not-ready-toleration-seconds` and `default-unreachable-toleration-seconds`. These parameters can each take values which let you control the reaction of the cluster to latency issues without needing to determine the best values using manual methods.
+
+[IMPORTANT]
+====
+Setting these parameters manually is not supported. Incorrect parameter settings adversely affect cluster stability.
+====
 
 All worker latency profiles configure the following parameters:
 
 --
-* `node-status-update-frequency`. Specifies the amount of time in seconds that a kubelet updates its status to the Kubernetes Controller Manager Operator.
-*  `node-monitor-grace-period`.  Specifies the amount of time in seconds that the Kubernetes Controller Manager Operator waits for an update from a kubelet before marking the node unhealthy and adding the `node.kubernetes.io/not-ready` or `node.kubernetes.io/unreachable` taint to the node.
-* `default-not-ready-toleration-seconds`. Specifies the amount of time in seconds after marking a node unhealthy that the Kubernetes Controller Manager Operator waits before evicting pods from that node. 
-* `default-unreachable-toleration-seconds`. Specifies the amount of time in seconds after marking a node unreachable that the Kubernetes Controller Manager Operator waits before evicting pods from that node.
+* `node-status-update-frequency`. Specifies the amount of time in seconds that a Node Controller (part of the KCM) updates its status by querying the API server.
```

Comment:
--node-status-update-frequency - Specifies how often kubelet posts its node status to the API server.

https://kubernetes.io/docs/concepts/architecture/nodes/